src/app/config/cla-env-utils.ts

```diff
@@ -0,0 +1,5 @@
+// Copyright The Linux Foundation and each contributor to CommunityBridge.
+// SPDX-License-Identifier: MIT
+
+import * as env from './cla-env-config.json';
```

Comment:
Renamed this file to be `vd-env-utils.ts`  change JSON to be: `vd-env-config.json`